llvm/test/CodeGen/AMDGPU/sdwa-peephole-instr-combine-sel-src.mir

```diff
@@ -0,0 +1,1089 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1030 -simplify-mir -run-pass=si-peephole-sdwa -o - %s | FileCheck %s
+
```

Comment:
I'm not sure I see the dest write cases?

Reply:
You are right, I missed to add the new file. Here it is now: https://github.com/llvm/llvm-project/pull/123221/commits/58e278fd77cc82527fd3f6ead618519283cfad9b#diff-30c0b74efc1ede6ed6f8915cdbfff782669645f27607e94a62a238ca82124624